src/main/java/seedu/address/logic/commands/FilterLocCommand.java

```diff
@@ -27,9 +27,10 @@ public FilterLocCommand(LocationContainsKeywordsPredicate predicate) {
     @Override
     public CommandResult execute(Model model) {
         requireNonNull(model);
-        model.updateFilteredPersonList(predicate);
```

Comment:
For this command can j ignore it cuz it has not been properly implemented with the updated stuff yet. Can just delete all filter commands if that makes things easier